### DISCUSSION TITLE
Distribution process so slow?

### DISCUSSION THREAD

**Discussion by hadmut** [puppetlabs/puppet-agent#2158] [General]
Hi, 

7.12.1 should contain an important bug fix. 

I wonder why it takes so long to get 7.12.1 to this repo, and why this repo indicates label 7.12.0, while 

gem search --remote -e puppet

*** REMOTE GEMS ***

puppet (7.11.0 ruby universal-darwin x64-mingw32 x86-mingw32)

still reports 7.11.0. and seem to be built from https://github.com/puppetlabs/puppet , which, again, has a 7.12.0 release. 

This is somewhat confusing. 

Unfortunately, not even the README.md do clarify, what repo contains what and how the gems are build from what.

**Comment by joshcooper** [comment]
7.12.1 was a security release so the changes were not pushed to our public repos until after the release. Normally, puppet and puppet-agent versions should match and be shipped together (to rubygems.org, yum.puppet.com, etc)
